js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -2444,16 +2444,16 @@ realm=Realm
 attributeConsumingServiceIndex=Attribute Consuming Service Index
 prompt=Prompt
 assign=Assign
-disableConfirmRealm=User and clients can't access the realm if it's disabled. Are you sure you want to continue?
+disableConfirmRealm=User and clients can't access the realm if it is disabled. Are you sure you want to continue?
 showAuthData=Show authorization data
 includeInUserInfo.tooltip=Should the claim be added to the userinfo?
 select=Select
 signature-algorithm=JWA algorithm, which the client needs to use when signing a JWT for authentication. If left blank, the client is allowed to use any appropriate algorithm for the particular client authenticator.
 advanced=Advanced
 initialCounter=Initial counter
-revokeRefreshTokenHelp=If enabled a refresh token can only be used up to 'Refresh Token Max Reuse' and is revoked when a different token is used. Otherwise refresh tokens are not revoked when used and can be used multiple times.
+revokeRefreshTokenHelp=If enabled, a refresh token can only be used up to 'Refresh Token Max Reuse' and is revoked when a different token is used. Otherwise refresh tokens are not revoked when used and can be used multiple times.
```

Comment:
```suggestion
revokeRefreshTokenHelp=If enabled, a refresh token can only be used up to 'Refresh Token Max Reuse' and it is revoked when a different token is used. Otherwise, refresh tokens are not revoked when used and can be used multiple times.
```